### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by rfdjadkin** [charlesmalafosse/sports-betting-customloss#3]
This doesn't work as advertised in the article. After fixing the datatype bug:

Training Loss : -0.16280347108840942
Validation Loss : 0.09591284394264221

I suppose you had a lucky run. Did you use a seed in training or anything else that could differ?
